tasks.yaml

```diff
@@ -34,61 +38,56 @@ tasks:
     description: "Build prereqs Zarf package"
     actions:
       - description: "Build package with RKE2 prereqs"
-        cmd: "./uds zarf p c -a ${ARCH} --confirm --no-progress --skip-sbom"
+        cmd: "./uds zarf package create -a ${ARCH} --confirm --no-progress --skip-sbom"
 
   - name: create-lima-vm
     description: "Creates Lima RKE2 VM"
     actions:
       - cmd: |
           if [[ "$(uname)" == "Darwin" ]]; then
-            limactl start template://experimental/rke2 \
-              --memory 20 --cpus 10 --vm-type=vz --network=vzNAT -y
+            limactl start --name="$VM_NAME" template://experimental/rke2 \
+              --vm-type=vz --network=vzNAT -y $LIMA_ARGS
           else
-            limactl start template://experimental/rke2 \
-              --memory 20 --cpus 10 --vm-type=qemu -y
+            limactl start --name="$VM_NAME" template://experimental/rke2 \
+              --vm-type=qemu -y $LIMA_ARGS
           fi
   - name: deploy
     description: "Deploys rke2-core-demo bundle"
     actions:
       - cmd: |
-          export KUBECONFIG=$(limactl list rke2 --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml')
+          export KUBECONFIG=$(limactl list --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml' -- "$VM_NAME")
           ./uds deploy uds-bundle-uds-rke2-demo-*.tar.zst --confirm
 
   - name: print-kubeconfig-path
     description: "Prints kubeconfig path for the RKE2 instance (Lima VM only)"
     actions: 
       - cmd: |
           echo  "****** Export the following env var to connect to cluster (Lima VM only!) ****** "
-          echo export KUBECONFIG=$(limactl list rke2 --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml')
+          echo export KUBECONFIG=$(limactl list --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml' -- "$VM_NAME")
           echo "\n"
 
   - name: delete-lima-vm
     description: "Deletes Lima VM"
     actions:
       - cmd: |
-          VM_NAME="rke2"
-
-          vm_line="$(limactl list 2>/dev/null | awk -v vm="$VM_NAME" '$1==vm {print}')"
-
-          if [ -z "${vm_line}" ]; then
+          if ! vm_status="$(limactl list --format '{{.Status}}' -- "${VM_NAME}" 2>/dev/null)"; then
             echo "VM '${VM_NAME}' does not exist."
             exit 0
           fi
 
-          # If the second column says Running, stop it
-          vm_status="$(echo "${vm_line}" | awk '{print $2}')"
           if [ "${vm_status}" = "Running" ]; then
-            echo "Stopping ${VM_NAME}..."
-            limactl stop "${VM_NAME}" || true
+            echo "Stopping '${VM_NAME}''..."
+            limactl stop -- "${VM_NAME}" || true
           fi
```

Comment:
I kept `limactl stop` because it was already here, but if you're deleting the VM anyway, is there any reason to stop it first? `limactl delete -f` works on a running VM.
```suggestion
```